ERROR  Cannot start nuxt:  Nuxt module should be a function: @nuxtjs/color-mode

When starting the dev server with nuxt 3.14.0 I get
ERROR  Cannot start nuxt:  Nuxt module should be a function: @nuxtjs/color-mode 

```
------------------------------
- Operating System: Windows_NT
- Node Version:     v22.11.0
- Nuxt Version:     3.14.0
- CLI Version:      3.15.0
- Nitro Version:    2.10.2
- Package Manager:  pnpm@9.12.3
- Builder:          -
- User Config:      default
- Runtime Modules:  normalizedModule(), normalizedModule(), @vueuse/nuxt@11.2.0, @nuxt/content@2.13.4, @nuxt/eslint@0.6.1, @nuxtjs/tailwindcss@6.12.2, @nuxt/devtools@1.6.0, shadcn-nuxt@0.11.2, nuxt-meilisearch@1.3.10, @nuxt/image@1.8.1, @formkit/nuxt@1.6.7, @nuxt/icon@1.6.1
- Build Modules:    -
------------------------------
```

Locally everything works, but in ci npm error

[nuxt-module-playground](https://github.com/GurovDmitriy/nuxt-module-playground)

Run npm ci
npm warn deprecated npmlog@5.0.1: This package is no longer supported.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated gauge@3.0.2: This package is no longer supported.
npm warn deprecated are-we-there-yet@2.0.0: This package is no longer supported.
npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v[4](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:5) are no longer supported
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated eslint@8.[5](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:6)7.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.
npm error code 1
npm error path /home/runner/work/nuxt-module-playground/nuxt-module-playground/packages/todo-landing
npm error command failed
npm error command sh -c nuxt prepare
Error: npm error [error] Nuxt module should be a function: @todo/todo-module
npm error   at loadNuxtModuleInstance (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/@nuxt/kit/dist/index.mjs:24[6](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:7)0:11)
npm error   at async installModule (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/@nuxt/kit/dist/index.mjs:2389:4[7](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:8))
npm error   at async initNuxt (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxt/dist/index.mjs:4669:5)
npm error   at async loadNuxt (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxt/dist/index.mjs:4[8](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:9)39:5)
npm error   at async loadNuxt (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/@nuxt/kit/dist/index.mjs:2627:1[9](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:10))
npm error   at async Object.run (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxi/dist/chunks/prepare.mjs:61:18)
npm error   at async runCommand$1 (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:1648:16)
npm error   at async runCommand$1 (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:1639:11)
npm error   at async runMain$1 (/home/runner/work/nuxt-module-playground/nuxt-module-playground/node_modules/nuxi/dist/shared/nuxi.bd0a2fa0.mjs:1777:7) 
npm error
Error: npm error [error] Nuxt module should be a function: @todo/todo-module
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2024-11-[10](https://github.com/GurovDmitriy/nuxt-module-playground/actions/runs/11765193334/job/32771230635#step:5:11)T12_38_14_131Z-debug-0.log
Error: Process completed with exit code 1.

--- a/packages/kit/src/module/install.ts
+++ b/packages/kit/src/module/install.ts
@@ -118,6 +118,11 @@ export async function loadNuxtModuleInstance (nuxtModule: string | NuxtModule, n
     }
   }
 
+  // Throw error if module could not be found
+  if (typeof nuxtModule === 'string') {
+    throw new TypeError(`Could not load \`${nuxtModule}\`. Is it installed?`)
+  }
+
   // Throw error if input is not a function
   if (typeof nuxtModule !== 'function') {
     throw new TypeError('Nuxt module should be a function: ' + nuxtModule)

